packages/README.md

```diff
@@ -0,0 +1,49 @@
+External Id Attestation
+-----------------------
+
+This feature allows a user to register an ID from an external system and given external verification the autority agent will make an attestation and associate the ID with a Holochain ``AgentPubKey``. Two additional fields (``display_name`` and ``request_id``) give the attestion object more uniqueness and traceability. A noticeble behaviour is that a user can register multiple external_id's against different AgentPubKeys. \
```

Comment:
"autority" should be "authority"

Reply:
fixed, sorry have added a spell checker extension to my IDE